distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -322,6 +322,9 @@ Provides a list of environment variables.
 |`spatial`
 |Searches spatially the local Catalog.
 
+|`validate`
+|Validates an XML file against all installed validators.
```

Comment:
I would say what the expected outcome is, something along the lines of "and prints out human readable errors and warnings or nothing if no validators fail"